solidity/test/StandardPoolConverter.js

```diff
@@ -1126,13 +1134,13 @@ describe('StandardPoolConverter', () => {
 
                 for (let i = 0; i < allowances.length; i++) {
                     const diff = Decimal(allowances[i].toString()).div(reserveAmounts[i].toString());
-                    expect(diff.eq('0')).to.be.true();
+                    expect(diff.toFixed()).to.be.equal('0');
```

Comment:
In general, we should really avoid mixing "numbers" and strings. In most of the cases (e.g., when precision isn't an issue), it's unneeded and complicates the code.

Reply:
Unfortunately, mocha doesn't have any builtin methods for `Decimal` (the type of variable `diff`).
My fix here is intended to output a more descriptive error-message when an error is thrown (since "expected false to be true" isn't a very descriptive error-message).

Reply:
My note is regarding `'0'`.

Reply:
So is mine.
Variable `diff` is of type `Decimal`.
We can either do the above, or we can do:
```
expect(diff.eq('0')).to.be.true();
```
Which is not any better, and in fact - worse, because it doesn't provide valuable information upon error.

Reply:
I'm not sure that I follow. Why wouldn't `expect(diff.toFixed()).to.be.equal(0);` work?

Reply:
`toFixed` returns a string, why compare it with a number?

Reply:
Oh, you're right. Let's hope that switching to Hardhat will make these irregularities disappear.